### PR TITLE
Add direction to Stop

### PR DESCRIPTION
**Pull request by DraXus**

### CODE REVIEW

File: src/Transport/Normalizers/StopNormalizer.php [hunk 1]
```diff
@@ -40,6 +41,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
         $stop->setCommonName($data['commonname']);
         $stop->setLocalityName($data['localityname']);
         $stop->setGeolocation(new Geolocation($data['lat'], $data['lng']));
+        $stop->setDirection($data['direction']);
```
Comment (dave1010) [inline src/Transport/Normalizers/StopNormalizer.php]: need to check property exists

Comment (DraXus) [inline src/Transport/Normalizers/StopNormalizer.php]: @dave1010 shall I check all the properties or only the direction?

**Comment by rossey** [comment]
:+1:

File: src/Transport/Entities/Stop.php [hunk 2]
```diff
@@ -31,53 +32,99 @@ public static function createFromArray(array $properties)
...
+    /**
+     * @param GeolocationInterface $geolocation
+     */
     public function setGeolocation(GeolocationInterface $geolocation)
     {
         $this->geolocation = $geolocation;
     }
 
+    /**
+     * @return string
```
Comment (rossey) [inline src/Transport/Entities/Stop.php]: This needs to return GeolocationInterface

**Pull request merged**